Make Overview less prescriptive (#23)
graphql/composite-schemas-spec: spec/Section 1 -- Overview.md

  specification facilitates the coordinated effort of combining collaboratively
  designed source schemas into a single coherent composite schema.

- **Evolvable**: A _composite schema_ enables offering an integrated,
  product-centric API interface to clients. Each source _GraphQL service_ that
  underpins the composite schema should be able to evolve without disrupting
  these clients. Over time, the same functionality may be provided by a

>   different combination of services, while the composite schemas interface can

Reviewer: I agree that "must" is too strong, but "can" is in my opinion too weak - GraphQL encourages versionless schema design, and a composite schema should be governed by this also. I suggest we change must/can to "should":

```suggestion
  different combination of services, while the composite schemas interface should
```

(this will require prettier again. Semantic line breaks is pretty neat - would be interesting to add that to prettier for sure!)

  continue to support existing requests from all clients; source schema

>   boundaries are therefore considered an implementation detail and should never

Reviewer: I think "should not" rather than "should never" is better here. I know this was in the original text, but while we're at it...

```suggestion
  boundaries are therefore considered an implementation detail and should not
```

> - **Evolvable**: A _composite schema_ enables offering an integrated,
>   product-centric API interface to clients. Each source _GraphQL service_ that
>   underpins the composite schema should be able to evolve without disrupting
>   these clients. Over time, the same functionality may be provided by a
>   different combination of services, while the composite schemas interface
>   should continue to support existing requests from all clients; source schema
>   boundaries are therefore considered an implementation detail and should not be
>   exposed to clients.

Reviewer: Maybe this is less controversial?

```suggestion
- **Evolvable**: A _composite schema_ enables offering an integrated,
  product-centric API interface to clients. Source schema boundaries are an
  implementation detail, a detail that is not exposed to clients. As each
  underlying _GraphQL service_ evolves, the same functionality may be provided
  from a different combination of services. This specification helps to ensure
  that changes to underlying services can be made whilst maintaining support for
  existing requests from all clients to the composite schema interface.
```
